mir-layout/src/main/resources/META-INF/resources/mir-layout/scss/common/bootstrap4fix.scss

```diff
@@ -32,21 +32,22 @@
 }
 .mcr-invalid {
   label {
-    color: red;
+    color: $danger;
   }
   input,
-  select {
-    border: 1px solid red;
-    padding-right: calc((1em + .75rem) * 3 / 4 + 1.75rem);
```

Comment:
is it a typo, that padding-right is now missing?

Reply:
As discussed in the ticket, that rule doesn't appear to affect anything. If it does, and I just didn't see it, that rule should probably also be present in the corresponding rule for `.mcr-valid`. Either way, dist is an inconsistency that should be addressed.